api/v1alpha1/release_webhook_test.go

```diff
@@ -45,42 +59,246 @@ var _ = Describe("Release validation webhook", func() {
 		}
 	})
 
-	AfterEach(func() {
-		_ = k8sClient.Delete(ctx, release)
-	})
+	Context("When a Release is created", func() {
+		BeforeEach(func() {
+			admissionRequest.AdmissionRequest.Operation = admissionv1.Create
+		})
+		It("should add admin as the value for the author label", func() {
+			admissionRequest.Object.Raw, err = json.Marshal(release)
+			Expect(err).NotTo(HaveOccurred())
+
+			rsp := rHandler.Handle(ctx, admissionRequest)
+			Expect(rsp.AdmissionResponse.UID).To(Equal(types.UID("123")))
+			Expect(rsp.AdmissionResponse.Allowed).To(BeTrue())
 
-	Context("Update Release CR fields", func() {
-		It("Should error out when updating the resource", func() {
-			ctx := context.Background()
+			var jsonPatch []jsonpatch.JsonPatchOperation
+			err = json.Unmarshal(rsp.AdmissionResponse.Patch, &jsonPatch)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(jsonPatch)).To(Equal(1))
+			patch := jsonPatch[0]
+			Expect(patch.Operation).To(Equal("replace"))
+			Expect(patch.Path).To(Equal("/metadata/labels"))
+			Expect(patch.Value).To(Equal(map[string]interface{}{
+				authorLabel: "admin",
+			}))
+		})
+		It("should overwrite the author label value when one is provided by user", func() {
```

Comment:
We should have empty lines between It blocks. Probably you inherited this. We have tests that need to be rewrote yet.

Reply:
done in follow up